doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -78,33 +124,3 @@ This has the following considerations when it comes to bare metal:
 Regarding the size of the nodes, this is about trying not to reduce the benefits we gained with NFV in the first place. For example, if three hypervisor hosts have 60CPUs and 1.5TB RAM between them, sharing those resources between a large number of applications (lets say 50) is made efficient by the use of virtualisation. For bare metal containerisation, that same number of applications would need a much larger number of bare metal hosts, with each host being a much smaller unit.
```

Comment:
Both containers in VMs and on bare metal have their use cases. That the number of application would generally need to be higher for bare metal hosts is not true; it depends on the resource consumption of the workloads, obviously. Virtualization has benefits for running multiple clusters on the same infra, but drawbacks (e.g. resource overhead and fragmentation) when running a single cluster. --> it all depends on the use case.

Reply:
+1
I would even argue the number of hosts required to run the same number of applications is usually smaller if you do it on bare metal, compared to running them on top of a CaaS in VMs
IMO the RA can define a CaaS with all the necessary features, and in a separate chapter / paper we could mention the many (at least two, but let's keep in mind we might need to have a "public cloud" discussion at some point :) ) ways of how one can deploy this reference CaaS